apps/nextjs/src/components/Onboarding/AcceptTermsForm.tsx

```diff
@@ -0,0 +1,138 @@
+"use client";
+
+import { useState } from "react";
+
+import { useUser } from "@clerk/nextjs";
+import logger from "@oakai/logger/browser";
+import {
+  OakBox,
+  OakFlex,
+  OakHeading,
+  OakLink,
+  OakP,
+  OakPrimaryButton,
+  OakSpan,
+} from "@oaknational/oak-components";
+import { useReloadSession } from "hooks/useReloadSession";
+import Link from "next/link";
+
+import Button from "@/components/Button";
+import CheckBox from "@/components/CheckBox";
+import SignUpSignInLayout from "@/components/SignUpSignInLayout";
+import TermsContent from "@/components/TermsContent";
+import { trpc } from "@/utils/trpc";
+
+export const AcceptTermsForm = () => {
+  const [dropDownOpen, setDropDownOpen] = useState(true);
+  const { isLoaded } = useUser();
+  const reloadSession = useReloadSession();
+
+  const [termsAcceptedLocal, setTermsAcceptedLocal] = useState(false);
+  const [privacyAcceptedLocal, setPrivacyAcceptedLocal] = useState(false);
+  const setDemoStatus = trpc.auth.setDemoStatus.useMutation();
+  const acceptTerms = trpc.auth.acceptTerms.useMutation({});
+
+  const handleAcceptTermsOfUse = async () => {
+    try {
+      await setDemoStatus.mutateAsync();
+      logger.debug("Demo status set successfully");
+
+      const response = await acceptTerms.mutateAsync({
+        termsOfUse: new Date(),
+        privacyPolicy: privacyAcceptedLocal ? new Date() : false,
```

Comment:
Do we want these two `accepted_at` Dates to be identical? Can't think of a reason when it would matter but feels like it would make sense

Reply:
Do you mean an identical instance, or do you mean that they should both always be set? `privacyPolicy` is a slight misnomer as it means "I want to receive marketing comms and have accepted the related privacy policy"

Reply:
Sorry i guess I meant the same instance (i.e. consent was given at exactly the same time)

Reply:
Ok. Will update this